stream-chat-android-client-sample/build.gradle

```diff
@@ -79,9 +79,7 @@ dependencies {
     kapt Dependencies.roomCompiler
 
     // Firebase
-    implementation platform(Dependencies.firebaseBom)
     implementation Dependencies.firebaseMessaging
-    implementation Dependencies.firebaseCore
```

Comment:
While I was fixing this, I also removed `firebase-core`, which is no longer needed.